### ISSUE TITLE
Syntax highlighter confused by generic arguments in function calls

### ISSUE THREAD

**Issue by kevinbarabash** [michaelgmcd/vscode-language-babel#57]
The syntax highlighter gets confused when it runs into a type parameter in a generic function call.

<img width="381" alt="Screen Shot 2019-09-01 at 6 51 58 PM" src="https://user-images.githubusercontent.com/1044413/64083208-b7867780-cce9-11e9-8c54-61e9b7eff7f5.png">

```js
type D = () => mixed;

const Foo = () => {
  const dispatch = useDispatch();

  return (
    <div>Hello, world!</div>
  )
};

const Bar = () => {
  const dispatch = useDispatch<D>();

  return (
    <div>Hello, world!</div>
  )
};
```

**Comment by mgtitimoli** [comment]
Duplicate of #43

**Comment by kevinbarabash** [comment]
That issue was closed by a PR.  Does there need to be a publish before that update will appear in the plugin in VSCode?

**Comment by michaelgmcd** [comment]
Please check that this was fixed in v0.0.24

**Comment by kevinbarabash** [comment]
@michaelgmcd it's fixed.  Thanks for publishing and thanks for writing this plugin.
